primitives/src/lib.rs

```diff
@@ -69,12 +72,13 @@ pub type SessionKey = ed25519::Public;
 /// that 32 bits may be multiplied with a balance in 128 bits without worrying about overflow.
 pub type Balance = u128;
 
+// TODO: I don't get why this was defined here. It was aura so I change it to babe but regardless of that
```

Comment:
yeah, I still agree with my note. _If the crypto key type is defined inside the `babe` module and exported via `babe::AuthorityId` then that should be used everywhere_. I don't get why the other one is needed.

Reply:
indeed.